entities.py

```diff
@@ -54,29 +55,29 @@ def __init__(self, name, pos, behavior):
         """
         super().__init__(name, pos, 'actor')
         self.behavior = behavior
-        # Give the behavior a reference to its owner entity
-        self.behavior.owner = self
+        self.char = char
+        self.color = color
         # base stats
         self._strength = 5
         self._constitution = 5
         self._intelligence = 5
         self._dexterity = 5
         self._luck = 5
+        # other stats
+        self._level = 1
+        self._exp = 0
+        self._max_exp = floor(1 + 300 * 2 ** (1/7)) / 4
+        self._gold = 100
         # computed stats
         self._max_hp = self._compute_max_hp()
         self._max_mp = self._compute_max_mp()
+        self._cur_hp = self._max_hp
+        self._cur_mp = self._max_mp
```

Comment:
note to self: undocumented change, perhaps push straight this straight to master as a fix and pull --rebase on this feature branch

Reply:
Yeah I had to fix the order in the testing branch too cus it was broken.